src/display/text_layer.js

```diff
@@ -406,10 +402,9 @@ class TextLayer {
       const { fontFamily } = style;
       const { canvasWidth, fontSize } = properties;
 
-      if (prevFontSize !== fontSize || prevFontFamily !== fontFamily) {
-        ctx.font = `${fontSize * this.#scale}px ${fontFamily}`;
-        params.prevFontSize = fontSize;
-        params.prevFontFamily = fontFamily;
```

Comment:
Also, please note why this code looks the way it does: To avoid needlessly querying the DOM to get the `ctx.font` data, when the font hasn't changed, for every single textLayer element.

Hence just removing this seems wrong, unfortunately (even if a test-case is provided).

Reply:
Yep and maybe I'm missing something but I don't understand how `#this.scale` can be changed while calling `#layout`.